backend/server.py

```
@@ -7620,6 +7620,26 @@ async def loops_overview() -> dict[str, object]:
         }
 
 
+@app.get("/api/cache/stats")
+async def response_cache_stats(user: dict = Depends(get_current_user)) -> dict[str, object]:
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)